process/sctfcoil.py

```diff
@@ -4255,7 +4255,7 @@ def stresscl(
         # -----------------------------
         # Array equation
         sig_tf_tresca_tmp1 = numpy.maximum(
-            numpy.absolute(sig_tf_r - sig_tf_t), numpy.absolute(sig_tf_r - sig_tf_z)
+            numpy.absolute(sig_tf_r), numpy.absolute(sig_tf_r - sig_tf_z)
         )
         sig_tf_tresca = numpy.maximum(
             sig_tf_tresca_tmp1, numpy.absolute(sig_tf_z - sig_tf_t)
```

Comment:
I am not sure why you have included $|\sigma_r|$.  It shouldn't make any difference because $\sigma_z$ should always be negative, but I don't think you need it.

So I think all you need is one line
```python
 sig_tf_tresca_tmp1 = numpy.maximum(
            numpy.absolute(sig_tf_z - sig_tf_t), numpy.absolute(sig_tf_r - sig_tf_z)
)
```